eventmesh-connector-plugin/eventmesh-connector-rocketmq/src/main/java/org/apache/eventmesh/connector/rocketmq/MessagingAccessPointImpl.java

```diff
@@ -52,7 +49,7 @@ public Properties attributes() {
 
     @Override
     public Producer createProducer(Properties properties) {
-        return new ProducerImpl(this.accessPointProperties);
+        return null;
```

Comment:
Why do this change?

Reply:
This class will be removed because OMS will be removed. This part is modified to avoid error reporting

Reply:
Does this change can keep the server start success?